app/assets/stylesheets/application.scss

```diff
@@ -25,6 +25,9 @@ body {
 .header {
   background-color: rgb(245, 153, 55);
   height: 100px;
+  .lang {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.